### DISCUSSION TITLE
Hosted Avatar images results build error

### DISCUSSION THREAD

**Discussion by gcafepgdi** [mearashadowfax/ScrewFast#201] [Q&A] [answered]
First and foremost, thank you so much for such a beautiful theme. 

I changed the code a little bit to test if it is possible to self host the avatar images in my own server instead of unsplash.com image; as shown below. It is now spitting the error "`Type 'ImageMetadata' is not assignable to type 'string'`." on build. There were no problem while testing in `npm run dev`. It doesn't build the static site on `npm run build`.

Any help?

```
// testimonials images
import test1 from "@images/reviews/_testimonial1.avif";
import test2 from "@images/reviews/_testimonial2.avif";
import test3 from "@images/reviews/_testimonial3.avif";
import test4 from "@images/reviews/_testimonial4.avif";

const avatarSrcs: Array<string> = [
  test4,
  test1,
  test2,
  test3,
];
```

**Comment by mearashadowfax** [comment, marked as answer]
Thank you for your kind words about the theme, @gcafepgdi! As for the issue you are facing, you might want to try using `const avatarSrcs: Array<any>` to bypass the type mismatch. Or you might want to look at this recipe: https://docs.astro.build/en/recipes/dynamically-importing-images/
